fluentui_drawer/src/main/java/com/microsoft/fluentui/persistentbottomsheet/PersistentBottomSheet.kt

```diff
@@ -279,6 +257,18 @@ class PersistentBottomSheet @JvmOverloads constructor(context: Context, attrs: A
         }
     }
 
+    private fun setClickListenerForDismissal() {
+        persistentSheetBinding.persistentBottomSheetOutlined.setOnClickListener {
+            if (persistentSheetBehavior.state == BottomSheetBehavior.STATE_EXPANDED) {
+                persistentSheetBehavior.state = BottomSheetBehavior.STATE_COLLAPSED
+            }
+            persistentSheetBinding.persistentBottomSheetOutlined.isClickable = false
+            if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Why are we doing this only for Android above O?
this can be moved to XML, is there is a limitation on API level

Reply:
When sheet is expanded then we want to absorb events, while it's collapse we don't want to. And as outlined is there, even after collapse, we set this to unfocusable in order to not capture events and pass them to below view. Setting focusable=false is required in order to not absorb events for >=O.